getting_started/intel_debug/suggestions.rst

```diff
@@ -135,8 +135,8 @@ file:
    options snd_intel_sdw_acpi dyndbg=+pmf
 
    # SOF internals
-   options snd_sof_intel_ipc dyndbg=+pmf
    options snd_sof_intel_hda_common dyndbg=+pmf
+   options snd_sof_intel_hda_generic dyndbg=+pmf
```

Comment:
Should we say from whence ? i.e. from 6.6+ do this else do that.

Reply:
I will separate the two change.
I cannot find any indication that we ever had `snd_sof_intel_ipc`